wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -1,123 +1,113 @@
-# Beat snap divisor
+# Beat Snap Divisor
 
-The Beat Snap Divisor is used to limit where notes can be placed so that they always fall on the beat. Moving the slider to the right increases the granularity of the notes, that is, more notes can be placed in the same measure.
+![Beat Snap Divisor in the editor](img/beat_snap_divisor.png "Beat Snap Divisor in the editor")
 
-Each of the settings put different colours of ticks on the [Timelines](/wiki/Beatmap_Editor/Timelines). The ticks are as follows (for [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch)):
+The **Beat Snap Divisor** is an editor setting that determines the editor's behaviour in quantising [hit objects](/wiki/Hit_Object) based on the [Beat Snap](/wiki/Beatmap_Editor/Beat_Snap) principles. It can be found in the upper-right portion of the editor screen, where it corresponds to the granularity of the objects on the [editor timeline](/wiki/Beatmap_Editor/Timelines); that is, a denser Beat Snap Divisor setting allows for more notes to be placed in the same measure and vice versa. 
```

Comment:
```suggestion
The **Beat Snap Divisor** is an editor setting that determines the editor's behaviour in quantising [hit objects](/wiki/Hit_object) based on the [Beat Snap](/wiki/Beatmap_Editor/Beat_Snap) principles. It can be found in the upper-right portion of the screen, where it corresponds to the granularity of the objects on the [timeline](/wiki/Beatmap_Editor/Timelines); that is, a denser Beat Snap Divisor setting allows for more notes to be placed in the same measure of time and vice versa. 
```